server/config/nl_page/chart_titles_by_sv.json

```diff
@@ -1118,10 +1124,12 @@
   "dc/2wgh04kx6es88": "Median Income for Population Working in the Health Care and Social Assistance Industry",
   "dc/34t2kjrwbjd31": "Population Working in the Petroleum and Coal Products Manufacturing Industry",
   "dc/3ex11eg2q9hp6": "Median Income of Manufacturing Workers",
+  "dc/3jr0p7yjw06z9": "BLS Gambling Establishments",
   "dc/3kwcvm428wpq4": "Rail Transportation Workers",
   "dc/4ky4sj05bw4nd": "Total Wages Earned by Population Working in the Air Transportation Industry",
   "dc/4mm2p1rxr5wz4": "Population Working in Miscellaneous Retail Stores",
   "dc/4z9xy6wn8xns1": "Median Income of Population Working in the Administrative and Support and Waste Management Services Industry",
+  "dc/5br285q68be6": "Population Working in Gambling Industries",
```

Comment:
This shows up as "Gambling Industries" and "Gambling Establishments".

Should we call this "Employees in Gambling Industry" (singular)?

Reply:
Done, updated and regenerated json